### DISCUSSION TITLE
How to start tracks to start one after the other

### DISCUSSION THREAD

**Discussion by 82un0** [gpx-animator/gpx-animator#560] [Q&A] [answered]
Hello,
First of all I wanted to say that I was very happy to find this project.
It's exactly what I was looking for. Well done!
I have a trip divided into several stages with a track for each stage. When I click on "render", the tracks start together.
I'd like the tracks to start one after the other, anyone got any ideas?

**Comment by UgoMolinari** [comment, marked as answer]
Crea filmati separati e poi uniscili con un programma di video editing.
Ciao

**Reply by 82un0** [reply]
È davvero molto semplice. Grazie mille

**Comment by McPringle** [comment]
Did you create the GPX files with a software while planning or have you recorded them with a navigation device or smartphone? That makes a huge difference because when you record a route with a navigation device or smartphone, it will add timestamps in the GPX file. You will not have these timestamps when you create the GPX files with a planning software. The GPX Animator uses these timestamps to automatically synchronize the animation of several tracks.

Example: A group of cyclists start at different times to meet at the same destination. If you add these GPX files to GPX Animator, it will synchronize the dates and time and start the animation with the rider who started first. All other riders will be added to the animation automatically in relation to their real difference in time.

This is how the resulting video will look like: https://cliptube.org/w/jC8aHi1mVecsUJz3ARrfE1

If your GPX files does not have the date and time information, GPX Animator can't do that calculation. In that case - or to adjust the date and time difference more to your personal needs - you can go to the track configuration and adjust the "Time Offset" value in the UI:

![image](https://github.com/gpx-animator/gpx-animator/assets/1254039/a187edcf-0fa4-4593-bcc8-27235babcd8a)

You can do the same on the command-line with the `--time-offset` argument. It will adjust the time of the selected GPX file. So use the "real time values". Example: If the cyclist started one hour later, adjust the value by 1 hour.

You can adjust the values in two directions: forward and backward. To adjust the date and time backwards, just use negative values.

If you have recorded GPX files and the cyclists did not start at the same time but you want it to appear in the animation at the same time, use the time offset to adjust it accordingly. In the future we might implement an a feature like "start all tracks at the same time" to calculate the time offset automatically (see: #361).
